packages/plugin-ext/src/api/plugin-api.ts

```diff
@@ -977,7 +977,7 @@ export interface DebugMain {
     $addBreakpoints(breakpoints: Breakpoint[]): Promise<void>;
     $removeBreakpoints(breakpoints: Breakpoint[]): Promise<void>;
     $startDebugging(folder: theia.WorkspaceFolder | undefined, nameOrConfiguration: string | theia.DebugConfiguration): Promise<boolean>;
-    $customRequest(command: string, args?: any): Promise<DebugProtocol.Response>;
+    $customRequest(sessionId: string, command: string, args?: any): Promise<DebugProtocol.Response>;
```

Comment:
shouldn't we add new method or add argument at the end to not break ?

Reply:
hmm is that in fact there were a problem, command was being sent instead of sessionId ?

Reply:
i thought it was a mistake in the first place, implementation has different method signature and was broken before by using `command` as `sessionId`:
https://github.com/theia-ide/theia/blob/438efefd9b90e53aa58f54f2a8dfd50af8ba045b/packages/plugin-ext/src/main/browser/debug/debug-main.ts#L196